src/_userflair.scss

```diff
@@ -17,16 +17,27 @@
 }
 
 // List service icons
-.flair:before {
+.flair::before {
 	content: "";
-	width: 16px;
-	height: 16px;
+	width: 0px;
+	height: 0px;
 	background: url("images/general.png") 0 0/128px;
 	border-radius: inherit;
 	margin-left: -16px;
 	transition: inherit;
 }
 
+.flair-MAL::before, 
+.flair-HB::before,
+.flair-ANI::before,
+.flair-AP::before,
+.flair-ann::before,
+.flair-anidb::before,
+.flair-custom::before {
```

Comment:
Unless you've already checked on staging and it looks fine, I think you also want to restrict the `margin-right` on `:hover` (set below) to these flair classes, otherwise you'll end up with empty space on the far left of the flair when hovering (0x0 elements/pseudos can still have margins).